cl/corpus_importer/tests.py

```diff
@@ -686,19 +685,21 @@ def test_short_opinion_matching(self) -> None:
         self.assertEqual(bad_match, 81)
 
     @patch(
-        "cl.corpus_importer.management.commands.harvard_opinions.filepath_list",
-        side_effect=[iglob(os.path.join(test_dir, "bankruptcy_case_*.json"))],
+        "cl.corpus_importer.management.commands.harvard_opinions.filepath_list"
     )
-    def test_bankruptcy_cases_no_flag(self, mock) -> None:
-        """Bankruptcy cases can be imported without flag?"""
-        self.assertSuccessfulParse(0)
-        print("Success, no bankruptcy case imported")
+    @patch("cl.corpus_importer.management.commands.harvard_opinions.read_json")
+    def test_bankruptcy_flag(self, read_json_func, filepath_list_func):
+        """Can we test bankruptcy court imports"""
+        filepath_list_func.return_value = ["/one/fake/filepath.json"]
+        court = CaseLawCourtFactory.create(
+            name="United States Bankruptcy Court for the Northern District of Alabama"
+        )
+        caselawdata = CaseLawFactory()
+        caselawdata["court"] = court
```

Comment:
I think you should be able to combine these three lines into something like:

```
caselawdata = CaseLawFactory(
    court=CaseLawCourtFactory.create(name='xyz')
)
```

I guess that's still three lines, but encapsulates things a bit more nicely?

Reply:
let me double check that - I think there was a reason it got separated